Calculator/Calculator/Model/CalculatorManager.swift

```diff
@@ -0,0 +1,141 @@
+//
+//  CalculatorManager.swift
+//  Calculator
+//
+//  Created by mint, Whales on 2023/06/15.
+//
+
+import Foundation
+
+struct CalculatorManager {
+    private var isZeroButtonUsed = true
+    private var isCalculated = false
+    private var formulasUntilNow = ""
+    
+    mutating func verifyButton(for number: String, currentLabel: String) -> String? {
```

Comment:
로직은 이해가 갑니다만, 정확하게 메서드명으로 동작을 설명하진 못하는거 같아요. 매개변수명도 사용하는 입장에서 조금 뜻이 모호합니다. 
다음과 같은 네이밍은 어떨까요?
```suggestion
    mutating func verifyNumber(_ currentInput: String, for accumulatedOperand: String) -> String? {
```
저도 영알못이라 정확하진 않지만, 그래도 어떤 동작을 하는지는 알 수 있죠?
메서드명에 버튼, 레이블과 같은 컴포넌트 이름보다는 그 컴포넌트가 나타내는 값으로 네이밍해주시는게 좀 더 가이드라인에 맞는 네이밍 방법입니다!

Reply:
메서드 로직을 직관적으로 설명하는 네이밍이라 너무 좋은 것 같아요!!!
추천 감사드립니다 원비!!